run-tests.py

```diff
@@ -66,6 +69,282 @@ def coverage(args):
             os.remove('.coverage')
 
 
+def doc_coverage(args):
+    """
+    Checks if the documentation covers all public classes and methods.
+    """
+    # Scan Myokit modules for classes and functions
+    modules, classes, functions = _doc_coverage_get_modules_classes_functions()
+
+    # Check if they're all in the index
+    ok = _doc_coverage_check_index(modules, classes, functions)
+
+    # Check if they're all shown somewhere
+    ok = ok and _doc_coverage_check_completeness(classes, functions)
+
+    # Terminate if failed
+    if not ok:
+        sys.exit(1)
+
+
+def _doc_coverage_check_completeness(classes, functions):
```

Comment:
@fcooper8472 do you mind if I include this adaptation of your cover-checking code in Myokit?